source/shared/core_conn.cpp

```diff
@@ -282,7 +362,8 @@ SQLRETURN core_odbc_connect( _Inout_ sqlsrv_conn* conn, _Inout_ std::string& con
         SQLCHAR state[ SQL_SQLSTATE_BUFSIZE ];
         SQLSMALLINT len;
         SQLRETURN sr = SQLGetDiagField( SQL_HANDLE_DBC, conn->handle(), 1, SQL_DIAG_SQLSTATE, state, SQL_SQLSTATE_BUFSIZE, &len );
-        // sql state IM002/IM003 in ODBC 17, means that the correct ODBC driver is not installed
+        // sql state IM002, means that the specified ODBC driver is not installed
+        // might return IM003 in ODBC 17 (an issue that will be fixed)
```

Comment:
clarify that this is an issue in ODBC 17 and in the preview version only.
And do we have a backlog item to address this later?

Reply:
Yes there's a backlog item for this and will be fixed.